# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/credentialmanagement/IdentityCredentialRequestManager.h
# Commit: da37e2d8a175
# Full Hash: da37e2d8a175fda08a98920d820dda0899b6aedf
# Author: Benjamin VanderSloot <bvandersloot@mozilla.com>
# Date: 2025-06-13 09:36:48
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251617
# ==============================================================================

diff -r 15b92f98f62c -r da37e2d8a175 toolkit/components/credentialmanagement/IdentityCredentialRequestManager.h
--- a/toolkit/components/credentialmanagement/IdentityCredentialRequestManager.h	Thu Jun 12 18:16:52 2025 +0000
+++ b/toolkit/components/credentialmanagement/IdentityCredentialRequestManager.h	Thu Jun 12 18:16:53 2025 +0000
@@ -25,7 +25,6 @@
   static StaticRefPtr<IdentityCredentialRequestManager> sSingleton;
   IdentityCredentialRequestManager() {};
   ~IdentityCredentialRequestManager() = default;
-
 };
 
 }  // namespace mozilla